src/interface/src/styleguide/treatment-expander/treatment-expander.component.ts

```diff
@@ -0,0 +1,121 @@
+import { Component, Input } from '@angular/core';
+import { NgIf, NgFor, NgClass } from '@angular/common';
+import { MatIconModule } from '@angular/material/icon';
+import { MatExpansionModule } from '@angular/material/expansion';
+import {
+  TreatmentTypeIconComponent,
+  TreatmentIconColor,
+} from '../treatment-type-icon/treatment-type-icon.component';
+import { SequenceIconComponent } from '../sequence-icon/sequence-icon.component';
+
+export interface rxType {
+  name: string;
+  year: number;
+}
+
+//TODO: This should live in a central place and possibly map to something that's not a string?
+export type TreatmentType =
```

Comment:
yeah this is not great, shouldn't live on this component...

Reply:
Sure, I thought it should move somewhere too -- should it go in treatment.types?

Reply:
yeah, as long as its not on the styleguide component itself I think its fine!
Im not loving the fact that we'll need to map this fully on the FE.... if we do, why do we even use the lookup endpoint?
Should we just use FE static code and save us some roundtrips?